src/helpers/getValidatedUntrustedAsset.ts

```diff
@@ -32,8 +33,8 @@ export const getValidatedUntrustedAsset = async ({
 
     asset = `${assetCode}:${homeDomainIssuer}`;
   } catch (e) {
-    log.error({ title: "Issuer domain error: ", body: e.message });
-    throw new Error(e.message);
+    log.error({ title: "Issuer domain error: ", body: getErrorMessage(e) });
+    throw new Error(e);
```

Comment:
Should you be passing Error a string? `throw new Error(getErrorMessage(e))`

Not sure what kind of errors `getIssuerFromDomain` might throw, but you might end up in wrapping something that's already an Error object in another Error object

Reply:
👍 You're right, I'll fix that. We have more control if we pass along the error string.